charts/sn-platform-detector/values.yaml

```diff
@@ -0,0 +1,81 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+## Namespace to deploy 
+namespace: ""
+stackName: pulsar-dector
```

Comment:
It should be `global.stackName` according to the `_common.tpl`. En..., I remembered it was just `stackName` without `global` in the `_common.tpl` when reviewing the PR for`sn-platform-vault`.

Reply:
yeah, changed to `global.stackName` so in sn-platform-stack we only need to define once instead of passing stack name to each subchart.
actually don't need stackName when using independently, removing it.